Updates to service map content on service node coloring logic (#28517)
DataDog/documentation: content/en/tracing/services/services_map.md

### Color

> Service node borders are colored with red or yellow depending on their respective Service Health state. [Service Health][7] consolidates signals across Watchdog anomalies, paging monitors, and incidents into one single health state for each service. Use Service Health to identify services that need immediate attention at a glance. 

Reviewer: ```suggestion
Service node borders are colored with red or yellow depending on their respective Service Health state. [Service Health][7] consolidates signals across Watchdog anomalies, paging monitors, and incidents into one single health state for each service. Use Service Health to quickly identify services that require immediate attention.
```
